all-recipes.md

```diff
@@ -2,8 +2,9 @@
 layout: page
 title: All Recipes
 ---
+{% assign sorted =  site.recipes | sort_natural: "title" %}
 <ul>
-{% for r in site.recipes %}
+{% for r in sorted %}
```

Comment:
Its prob not really important, but insuring the recipes are sorted by title, just like the sidebar.